TypeApprovalRegister.yaml

```diff
@@ -232,12 +232,12 @@ paths:
                           type: string
                           description: >
                             'Name of application that shall be subject to the status record
-                            from [/core-model-1-4:control-construct/profile-collection/profile=tar-1-0-0-application-p-*/application-profile-1-0:application-profile-pac/application-profile-capability/application-name]'
+                            from [/core-model-1-4:control-construct/profile-collection/profile=tar-2-0-0-application-p-*/application-profile-1-0:application-profile-pac/application-profile-capability/application-name]'
```

Comment:
Sorry for stressing this point again. 
    The existing descriptions of attributes ( for services mentioned in above [comment](https://github.com/openBackhaul/TypeApprovalRegister/pull/75#discussion_r1026080793)) points to application-profile instances which has become obsolete after including file-profile to architecture and since application-data is included in an external-file. I think there would not be any clash with https://github.com/openBackhaul/TypeApprovalRegister/issues/69 while changing the above description(s) since the new descriptions would completely eliminate the concept of uuid. 
Please let me know if I am wrong. 
Or please let me know if we can create a separate issue for this and update descriptions there as https://github.com/openBackhaul/TypeApprovalRegister/issues/69 is related to uuid whereas this comment is related to concept of including file-profile

Reply:
I do not want to have the mess coming from two branches changing the same line of code.
I decided for updating the ServiceUUIDs and the Profiles in parallel, because activities are affecting different files, respectively different sections of the OAS.
Please, create a separate Issue, which is covering the ProfileUUIDs.

Reply:
Thanks for clarification. 
I have created a separate issue for updating descriptions related to application-data attributes. We can find it in https://github.com/openBackhaul/TypeApprovalRegister/issues/77

Reply:
Many thanks.